README.md

```diff
@@ -1,71 +1,34 @@
-# event-router-service
+## event-router-service
 
-Event router service routes the events from Bhamni to external system and vice versa. 
-As of now it supports routing the message from Bahmni to GCP Pub sub and GCP pub sub to Bahmni.
+Event router service routes the events from Bhamni to external system and vice versa.
+As of now it supports routing the message from Bahmni to GCP Pub sub.
 Configuration has to be provided both as an environment variable and through configuration files.
+Note: We expect the message to be received and published in JSON format.
 
-Subscription from bahmni
+## Packaging
+```mvn clean package```
 
-Run test using below command:
-mvn clean install
+## Prerequisite
+    JDK 1.17
```

Comment:
The event router service is an independent microservice. Why does it run on such an old JDK version?

Reply:
Sorry. My mistake. I read this as 7!